pyproject.toml

```diff
@@ -94,6 +94,8 @@ dev = [
     "types-psutil==5.8.23",
     "types_requests==2.31.0.2",
     "types-protobuf==4.23.0.3",
+    "yaramod==3.20.1",
+    "mkYARA==1.0.0",
```

Comment:
should we add a new entry here like `scripts = [ ...` or even more specific `match2yara`?

Reply:
Could do, but that will require adding a step to the tests github workflow  to install the scripts dependencies.

Is that ok?

Reply:
Updated

Reply:
great, thanks!